test/number/format-decimal.js

```diff
@@ -152,6 +152,19 @@ describe('NumberFormat', function() {
 				expect(value).to.equal('10,00000');
 			});
 
+			it('should handle group size of 0', function() {
+				localeData.groupSize = 0;
+				var options = {maximumFractionDigits: 2};
+				var value = format(1000000.01, localeData, options);
+				expect(value).to.equal('1000000.01');
+			});
+
+			it('should handle variable group sizes', function() {
```

Comment:
I'm not sure this test case it really testing the behaviour of the "0" value, which should stop further grouping from occurring. A more thorough input/output might be: input = "1000000000.01", output = "10000,00,000.01"